cli/cmd/release_lint.go

```diff
@@ -1,29 +1,45 @@
 package cmd
 
 import (
-	"fmt"
 	"github.com/pkg/errors"
 	"github.com/replicatedhq/replicated/cli/print"
+	"github.com/replicatedhq/replicated/pkg/types"
 	"github.com/spf13/cobra"
 )
 
+
+var (
+	validFailOnValues = map[string]interface{}{
+		"error": nil,
+		"warn": nil,
+		"info": nil,
+		"none": nil,
+		"": nil,
+	}
+)
+
 func (r *runners) InitReleaseLint(parent *cobra.Command) {
 	cmd := &cobra.Command{
-		Use:   "lint",
-		Short: "Lint a directory of KOTS manifests",
-		Long: "Lint a directory of KOTS manifests",
+		Use:          "lint",
+		Short:        "Lint a directory of KOTS manifests",
+		Long:         "Lint a directory of KOTS manifests",
 		SilenceUsage: true,
 	}
 	parent.AddCommand(cmd)
 
 	cmd.Flags().StringVar(&r.args.lintReleaseYamlDir, "yaml-dir", "", "The directory containing multiple yamls for a Kots release.  Cannot be used with the `yaml` flag.")
+	cmd.Flags().StringVar(&r.args.lintReleaseFailOn, "fail-on", "error", "The minimum severity to cause the command to exit with a non-zero exit code. Supported values are [info, warn, error, none].")
```

Comment:
I think the third param is for the type, and not the default value. As such, I think "error" should be changed to "string." Also, possibly, the third param for yaml-dir should be "path."